packages/runtime/src/interpreter/opcode.ts

```diff
@@ -176,4 +198,19 @@ export class Op {
     }
     return stack;
   }
+
+  /**
+   * Parses binary string into bigint
+   * @param binary Binary string array
+   */
+  parseToBigInt (binary: string[]): bigint {
```

Comment:
Can we also update this function to take a string as well? Eg. `parseToBigInt('101') => 5n`

Reply:
Also `parseToBigInt` sounds a bit ambigous. Maybe we could rename to `parseBinaryStrToBigInt` ?

Reply:
yes